.github/workflows/release-docker.yml

```diff
@@ -0,0 +1,27 @@
+name: Release Docker Image
+
+on: 
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    name: Docker build and push
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Get version without v character
+        id: version
+        run: |
+          VERSION=${{github.event.release.tag_name}}
+          VERSION_WITHOUT_V=${VERSION:1}
+          echo "::set-output name=value::$(echo $VERSION_WITHOUT_V)"
+      - name: Release to Docker
```

Comment:
Shouldnt we also follow conventional commits for a go/not go decision on releasing?

I did this on the `event-gateway` https://github.com/asyncapi/event-gateway/blob/master/.github/workflows/release.yml#L50-L64

Reply:
Hmm, interesting 🤔 , does that script also push `latest` and semver tag?

Reply:
Yupm it does.